app/views/credit_cards/_card_home_page.erb

```diff
@@ -0,0 +1,13 @@
+<div class= "row justify-content-center py-3">
```

Comment:
You just need the `<div class="credit-card">` inside the component. I suggest you remove the first 2 lines and the last one.